### DISCUSSION TITLE
Fractions not showing on epubs

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#2794] [Q&A]
When browsing epubs some fractions are not showing up and being replaced with two square brackets forming a square like []

**Comment by GitHubRulesOK** [comment]
common problem with fonts that expect 1/8 as normal text, but without sample no way to see why and 
any potential fix will likely depend on MuPDF library updates 
so always test most recent pre-release https://www.sumatrapdfreader.org/prerelease

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/45445316/178139108-49d00808-81c1-4942-bf03-617300724f5f.png)

I hope this is what you meant by a sample, as you can see, only the fraction at the top isn't showing, whilst one below is

**Reply by tactilis** [reply]
@martync1234
> I hope this is what you meant by a sample

Upload the PDF (not an screenshot image), or a link to where the PDF can be downloaded

**Comment by ghost** [comment]
Ah apologies, here you go 

https://drive.google.com/file/d/1SdMJAYQOePdw9LVceKFHC13N1MJFF4hO/view?usp=sharing

**Comment by tactilis** [comment]
If I select & copy the text from the PDF and paste it into Word, highlight the fraction and then look at Symbols, I see:

![image](https://user-images.githubusercontent.com/4060092/178156922-4a93155d-b166-471d-8648-f1f9d5d27545.png)

**Reply by GitHubRulesOK** [reply]
Odd I thought the online source shows 1/2 so perhaps some error "½ quantity of char siu roast pork (see above)" https://www.stylist.co.uk/food-drink/best-bao-buns-recipes/441859

**Reply by tactilis** [reply]
You are correct, but the .epub says 1/3 and that's what's not displaying in SumatraPDF

**Reply by GitHubRulesOK** [reply]
agreed it cant display 4 x 1/3 🙂 
![image](https://user-images.githubusercontent.com/12754913/178159083-3ade31c0-86e5-42ca-9368-fe019a10646d.png)

**Comment by tactilis** [comment]
Notepad++ shows line 67 of chapter01.xhtml in the .epub as:

![image](https://user-images.githubusercontent.com/4060092/178157335-631903f9-81e5-49ec-9530-e06d0922aecf.png)
